docs/administration-guide/19-formatting-settings.md

```diff
@@ -0,0 +1,51 @@
+## Setting default formatting
+
+There are lots of Metabase users around the world, each with different preferences for how dates, times, numbers, and currencies should be formatted and displayed. Metabase allows you to customize these formatting options at three different levels:
+
+1. Global formatting defaults in the Admin Panel
+2. Field-level formatting overrides in the Data Model section
+3. Question-level overrides in visualization settings
+
+### Global formatting defaults
+Here are the formatting options available to you from the `Formatting` tab of the `Settings` section in the Admin Panel:
+
+**Dates and Times**
+* `Date style:` the way dates should be displayed in tables, axis labels, and tooltips.
+* `Date separators:` you can choose between slashes, dashes, and dots here.
+* `Abbreviate names of days and months:` whenever a date is displayed with the day of the week and/or the month written out, turning this setting on will display e.g. `January` as `Jan` or `Monday` as `Mon`.
+* `Time style:` this lets you choose between a 12-hour or 24-hour clock to display the time by default where applicable.
+
+**Numbers**
+* `Separator style:` some folks use commas to separate thousands places, and others use periods. Here's where you can indicate which camp you belong to.
+
+**Currency**
+* `Unit of currency:` if you do most of your business in a particular currency, you can specify that here.
```

Comment:
Completely unrelated to documentation but I want a "Doing business" spinner in the app somewhere now :D